config/blobs.yml

```diff
@@ -38,6 +38,10 @@ golang/go1.8.3.linux-amd64.tar.gz:
   size: 90029041
   object_id: 531dbe5c-f65b-4c3a-a524-c630a7762567
   sha: 838c415896ef5ecd395dfabde5e7e6f8ac943c8e
+golang/go1.11.4.linux-amd64.tar.gz:
+  size: 126643341
+  object_id: b399f06d-3af0-4e3f-7103-41c0446f6ef6
+  sha: 52bd8eb353a39138a2ddf053857a0fd13f1f176c
```

Comment:
So we now use 3 different versions of go !!

Reply:
Yes, usually the recommendation is to keep them in the blobs.yml so that earlier versions continue to work. Hence keeping all of them.